### DISCUSSION TITLE
小程序的lite 版本上显示很模糊

### DISCUSSION THREAD

**Discussion by jackeryjam** [Tencent/libpag#1713] [Q&A]
之前是用完整版的， 在小程序上运行会有卡顿问题， 之后就换成lite版本， 不在卡顿了，但是在小程序上显示却很模糊
素材如下，
https://gwg.jackjam.xyz/cicc/1018/%E6%96%B0%E9%A6%96%E9%A1%B5_bmp.pag

用web版的lite显示也很正常， 很清晰， 但是小程序里面就很模糊，求助

**Comment by jackeryjam** [comment]
有没有大佬指点一下， 太难搞了

**Reply by jackeryjam** [reply]
急

**Comment by domchen** [comment]
截图对比发一下，有demo的话也上传一下。这样一句话的问题我们也办不了你。

**Reply by jackeryjam** [reply]
https://developers.weixin.qq.com/s/DYD0OUmw7tLP
这个是lite版本下的小程序demo，表现很模糊

下面这个是使用lite的web版本， 对比起来会发现挺清晰的
https://gwg.jackjam.xyz/tmp/pag-test/index4.html

这是pag文件 https://gwg.jackjam.xyz/tmp/pag-test/open.pag

**Reply by jackeryjam** [reply]
lite模式下是会有些模糊，换成完整版本是好的，当时换成lite版本的原因是，完整版多次播放之后动画会卡顿

https://developers.weixin.qq.com/s/bYLEdOmF7iLS
这个是小程序的demo，大佬有空帮忙看下这个动画，要开启调试模式才能看到那个动画，大概循环个五六次后就会开始卡顿了，我的手机是ios

**Reply by jackeryjam** [reply]
辛苦大佬看看哈， 我们那次活动已经上线了，没用成pag，但是发现pag真的是个很棒的东西，所以就过来请教大佬了哈

**Reply by jackeryjam** [reply]
请问一下这个有看出是啥问题么

**Comment by jackeryjam** [comment]
请问一下这个有看出是啥问题么

**Comment by jorxie** [comment]
请问这个问题解决了吗

**Reply by jinwuwu001** [reply]
上述 demo 中使用 pag 文件的链接已经无法访问了，请上传一份可复现问题的 demo，近期我们排查下该问题
